vm/systemSmartContracts/delegation.go

```diff
@@ -958,12 +960,61 @@ func (d *delegation) changeOwner(args *vmcommon.ContractCallInput) vmcommon.Retu
 	d.eei.SetStorageForAddress(d.delegationMgrSCAddress, args.Arguments[0], args.RecipientAddr)
 	d.eei.SetStorageForAddress(d.delegationMgrSCAddress, args.CallerAddr, []byte{})
 	d.eei.SetStorage([]byte(ownerKey), args.Arguments[0])
+	shouldUpdateAlsoTheAccount := d.enableEpochsHandler.IsMultiClaimOnDelegationEnabled()
```

Comment:
make a new function for these new lines - and treat the epoch there.

Reply:
yup. It will be clear in case we will keep using the same flag. Another option is to rename the flag to state it is used on the claim multi feat & this fix.

Reply:
added flag & removed the intermediate variable.